config/initializers/openstax_accounts.rb

```diff
@@ -0,0 +1,6 @@
+OpenStax::Accounts.configure do |config|
+  config.openstax_application_id = '1e0796d11bd4d92339c6a2f41c314440e90a7df91f302bbf332914c823c2d3d8'
```

Comment:
these probably shouldn't be checked in

Reply:
@mwharrison Moved to secrets. I deleted and recreated the oauth app.